src/cluster.py

```diff
@@ -71,6 +77,30 @@ def bootstrap_cluster(self, password: str) -> None:
             if not self._is_cluster_running():
                 raise ClusterNotRunningError()
 
+    def _change_owner(self, path: str) -> None:
+        """Change the ownership of a file or a directory to the postgres user.
+
+        Args:
+            path: path to a file or directory.
+        """
+        # Get the uid/gid for the postgres user.
+        u = pwd.getpwnam("postgres")
```

Comment:
thoughts on renaming `u` to `user_database`?

Reply:
It seems better. I changed it.